netdeploy/remote/nodecfg/nodeDir.go

```diff
@@ -143,7 +143,7 @@ func (nd *nodeDir) configureNetAddress(address string) (err error) {
 	}
 	fmt.Fprintf(os.Stdout, " - Assigning NetAddress: %s\n", address)
 	nd.config.NetAddress = address
-	if address != "" && address[0] == ':' && nd.IsRelay {
+	if address != "" && address[0] == ':' && nd.IsRelay() {
```

Comment:
You can remove the call to nd.IsRelay() - it's redundant now.

Reply:
redundancy eliminated, but kept nd.IsRelay() because that has some meaning